### PR TITLE
add discount dropdown and renewal -5%

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

File: src/components/PaymentLog.vue [hunk 1]
```diff
@@ -514,10 +550,24 @@ export default {
...
     },
+    async getDiscounts() {
+      try {
+        const fetchDiscounts = await get(
+          this.apiUrls.discounts
+        );
+        this.discounts = fetchDiscounts.data.data.data
+        this.discounts = this.discounts.filter((item)=> {
+          return item.id == 6 || item.id == 1});
+        console.log(this.discounts)
```
Comment (teebarg) [inline src/components/PaymentLog.vue:561]: remove log

File: src/components/PaymentLog.vue [hunk 2]
```diff
@@ -514,10 +550,24 @@ export default {
...
       }
     },
+    async getDiscounts() {
+      try {
+        const fetchDiscounts = await get(
+          this.apiUrls.discounts
+        );
+        this.discounts = fetchDiscounts.data.data.data
+        this.discounts = this.discounts.filter((item)=> {
+          return item.id == 6 || item.id == 1});
```
Comment (teebarg) [inline src/components/PaymentLog.vue:560]: what ids are these?

File: src/components/PaymentLog.vue [hunk 3]
```diff
@@ -402,6 +429,13 @@ export default {
     }
   },
   methods: {
+    watchSalesLogForm(){
+      if(this.salesLogForm.sales_category_id == "2"){
```
Comment (teebarg) [inline src/components/PaymentLog.vue:433]: can't we use name?

File: src/utilities/calculator.js [hunk 4]
```diff
@@ -17,8 +17,14 @@ const calculate = (productPrice, data, params) => {
   const tempActualRepayment = +(total - downpayment).toFixed(2);
   var   biMonthlyRepayment = Math.round((tempActualRepayment/count)/100)*100
   const actualRepayment = biMonthlyRepayment * count;
+  if(data.sales_category_id == 2 && data.discount_id == 6 ){
```
Comment (teebarg) [inline src/utilities/calculator.js:20]: what if id changes?

File: src/utilities/calculator.js [hunk 5]
```diff
@@ -17,8 +17,14 @@ const calculate = (productPrice, data, params) => {
...
+        var   rePayment = actualRepayment - (actualRepayment * 0.05)
```
Comment (teebarg) [inline src/utilities/calculator.js:21]: why are we hard coding discount?

File: src/components/PaymentLog.vue (hunk 1 above)
Comment (cheedinmar) [reply src/components/PaymentLog.vue:561]: ok done

File: src/components/PaymentLog.vue (hunk 2 above)
Comment (cheedinmar) [reply src/components/PaymentLog.vue:560]: new sales & renewal. But I checked now, you have changed the ids

File: src/components/PaymentLog.vue (hunk 3 above)
Comment (cheedinmar) [reply src/components/PaymentLog.vue:433]: ok done

File: src/components/PaymentLog.vue (hunk 2 above)
Comment (cheedinmar) [reply src/components/PaymentLog.vue]: I have removed it

File: src/utilities/calculator.js (hunk 5 above)
Comment (cheedinmar) [reply src/utilities/calculator.js]: resolved

File: src/components/PaymentLog.vue [hunk 6]
```diff
@@ -402,6 +437,13 @@ export default {
     }
   },
   methods: {
+    watchSalesLogForm(){
+      if(this.salesLogForm.sales_category_id == "2" && this.flag == 'beta'){
```
Comment (teebarg) [inline src/components/PaymentLog.vue:441]: check if renewal sales on live has id 2 as well

Comment (cheedinmar) [reply src/components/PaymentLog.vue:441]: ok

**Review by teebarg — APPROVED** [review]

**Pull request merged**